src/macros.rs

```diff
@@ -30,7 +30,7 @@
 #[macro_export(local_inner_macros)]
 macro_rules! log {
     // log!(target: "my_target", Level::Info; key1 = 42, key2 = true; "a {} event", "log");
-    (target: $target:expr, $lvl:expr, $($key:ident = $value:expr),+; $($arg:tt)+) => ({
+    (target: $target:expr, $lvl:expr, $($key:tt = $value:expr),+; $($arg:tt)+) => ({
```

Comment:
Ahh unfortunately I think this will include quotes in string keys. So for `"string/key"` the actual string we get will look like `"\"string/key\""`.

Looking at this again though, I don't think we actually need a tt muncher yet afterall. We can just use another macro around `__log_stringify!` (or just re-use `__log_stringify!` for it) that matches on literals and idents and handles them appropriately.